src/core/core.controller.js

```diff
@@ -867,10 +867,11 @@ module.exports = function(Chart) {
 			plugins.notify(me, 'afterEvent', [e]);
 
 			var bufferedRequest = me._bufferedRequest;
+			var valueOrDefault = helpers.valueOrDefault;
 			if (bufferedRequest) {
 				// If we have an update that was triggered, we need to do a normal render
 				me.render(bufferedRequest);
-			} else if (changed && !me.animating) {
+			} else if (changed && !me.animating && (valueOrDefault(me.options.tooltips.enabled, true) || valueOrDefault(me.options.hover.enabled, true))) {
```

Comment:
I'm not sure that's the correct approach: the `changed` variable should be used to detect these cases, so I would move that logic in the [tooltip event handler](/chartjs/Chart.js/blob/master/src/core/core.tooltip.js#L843) and return `false` if disabled:

```javascript
handleEvent: function(e) {
    var me = this;
    var changed = false;
    var options = me._options;

    if (!options || options.enabled === false) {
        return changed ;
    }

    // ...
```

Note that `chart.options.tooltips` can be `false`.

Moving that logic in tooltip, you don't anymore need to care of the `hover` options.